pkg/ccl/backupccl/backup_planning.go

```diff
@@ -1204,25 +1204,62 @@ func makeNewEncryptionOptions(
 	return encryptionOptions, encryptionInfo, nil
 }
 
+func getProtectedTimestampTargetForBackup(backupManifest BackupManifest) *ptpb.Target {
+	if backupManifest.DescriptorCoverage == tree.AllDescriptors {
+		return ptpb.MakeClusterTarget()
+	}
+
+	if len(backupManifest.Tenants) > 0 {
+		tenantID := make([]roachpb.TenantID, 0)
+		for _, tenant := range backupManifest.Tenants {
+			tenantID = append(tenantID, roachpb.MakeTenantID(tenant.ID))
+		}
+		return ptpb.MakeTenantsTarget(tenantID)
```

Comment:
Currently BACKUP syntax doesn't allow backing up table 245 and tenant 123, but that _is_ a valid backup job. It looks like however that is not possible with the ptpb.Target? i.e. tenants target and schema object targets are mutually exclusive?

Not blocking, since it's unreachable with current syntax, just wondering.

Reply:
It's not possible today yeah, we could in the future switch the Record to hold a `repeated Target` instead of a single target to accommodate for this.